middlewares/inputValidation.js

```diff
@@ -1,4 +1,4 @@
-import { check, validationResult } from 'express-validator';
+import { check, validationResult } from 'express-validator/check';
```

Comment:
Do you have the latest changes from `develop`? Your copy of this file seem to be outdated

Reply:
I had the latest develop as at that time